ntbs-service/wwwroot/css/home.scss

```diff
@@ -0,0 +1,23 @@
+.alerts-heading {
+    height: 100%;
+    margin: auto 0;
+}
+
+.tb-service-alerts-filter-label {
+    height: 100%;
+    margin: auto 0 auto auto;
+}
+
+.tb-service-alerts-filter-dropdown {
+    max-width: 200px;
+    margin: 5px 0 auto 15px
+}
+
+.reset-button-styling {
+    background: none;
+    border: none;
+    color: #003078;
+    text-decoration: underline;
+    cursor: pointer;
+    font-size: large;
+}
```

Comment:
Do we need this? I  recall we added a reset sheet in the past?

Reply:
Unless this wants to be called `link-button`, in which case it will probably make sense in the main stylesheet :)

Reply:
Yeah the reset didn't reset all of the inherent button styling  - I'll add it as a link-button class to the main sheet :)